modules/socialApi.js

```diff
@@ -0,0 +1,74 @@
+// socialApi.js
+
+const apiAddress = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+const likesUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4jsLQspdbzdaBFGyapdL/likes';
+const unqKey = '4jsLQspdbzdaBFGyapdL';
+
+const pushCommentToApi = async (pushThisToApi) => {
```

Comment:
- [ ] As requested by the previous reviewer, kindly implement the comment pop-up because it is required in the project. I know you said you worked solo which is understandable but then we have to follow the code review guideline and before I requested these changes, I confirmed with a senior code reviewer. You can write to student success about this or request a second opinion review.

Reply:
Thank you, I have added comment popup modal